book/website/_toc.yml

```diff
@@ -365,7 +367,7 @@
       file: community-handbook/consistency/consistency-structure
     - title: Language
       file: community-handbook/consistency/consistency-language
-  - title: Contributing and Developing Chapters - Templates and Workflow
+  - title: Chapter Templates and Contribution Workflow
```

Comment:
I think this edit should be in a separate pull request/issue, but I am not sure.